README.md

```diff
@@ -58,7 +58,7 @@ let SomeComponent = props => {
 
 ### Do I Need To Use the Babel Plugin?
 
-The babel plugin is not required, but enables some optimizations and customizations that could be beneficial for your project.
+The babel plugin is not required, but it removes the need for both the `/** @jsx jsx */` comment and the unused jsx import, and enables some optimizations and customizations that could be beneficial for your project.
```

Comment:
This isn’t true of the babel plugin, it’s only not necessary when using @emotion/babel-preset-css-prop.

Reply:
Ah good catch
Since the preset includes the plugin, would it make sense to replace this line to reference the preset instead?

Reply:
Maybe something like ![image](https://user-images.githubusercontent.com/743976/53099362-a5827b80-34f3-11e9-92d9-16533e07d8bc.png) ? 

Still linking to the plugin because the readme for the preset doesn't contain the feature table. Perhaps it'd make sense to extract that out into a separate md file?